apps/google-analytics-4/lambda/package.json

```diff
@@ -5,13 +5,14 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "debug": "NODE_ENV=development CIRCLE_SHA1=$(git rev-parse --short HEAD) SLS_DEBUG=* node inspect ./node_modules/serverless/bin/serverless.js offline",
-    "build": "NODE_ENV=production CIRCLE_SHA1=$(git rev-parse --short HEAD) rimraf ./build && tsc",
+    "debug": "NODE_ENV=development SLS_DEBUG=* node inspect ./node_modules/serverless/bin/serverless.js offline",
+    "build": "NODE_ENV=production rimraf ./build && tsc",
```

Comment:
why was the `CIRCLE_SHA1` removed here?

Reply:
Good question. i _believe_ we don't need this env var during the debug or build modes, but only during the actual deploy step. We fall back to "undefined" in our template serverless.yml. So removing it to get rid of cruft in our package.json file.

Do we need it or is it desirable? We can find a way to include it if so.

Reply:
I _believe_ that it needs to be included in the build otherwise the compiled [frontend] files will have an `undefined` sentry release configured